README.md

```diff
@@ -6,3 +6,10 @@ The goal is to develop a demo operator with level 5 capabilities to serve as an
 
 - The L5 Operator requires ingress controller to be installed if running in k8s cluster.
 - Steps for installing ingress controller for different clusters can be followed from [here](https://kubernetes.github.io/ingress-nginx/deploy/)
+  
+# Installation
+
+#### Make sure to install Crunchy Data Postgres Operator before installing the L5 operator
+
+- Crunchy Data Postgres Operator can be directly installed from the [Openshift Container Platform](https://oauth-openshift.apps.eng.opdev.io/oauth/authorize?client_id=console&redirect_uri=https%3A%2F%2Fconsole-openshift-console.apps.eng.opdev.io%2Fauth%2Fcallback&response_type=code&scope=user%3Afull&state=d933ad0d), when running the application on an Openshift Cluster.
```

Comment:
you should not be linking the team cluster here..

Reply:
the manual install of the postgres operator is only needed on vanilla kubernetes when installing without OLM..
May be best to list all methods
make deploy && make run (requires manual pgo install)
run bundle.. (if olm is available)
operator hub (once published as a community operator)
.. and have separate notes for vanilla k8s which requires ingress (i think usually always there ?) and pgo to be installed manually as well..
may be .. (edited)